app/state/reducers/searchFiltersReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+  search: '',
+};
+
+function searchFiltersReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'ENTER_OR_CHANGE_SEARCH_PAGE': {
```

Comment:
I would prefer keeping the { } to [create blocks](https://github.com/airbnb/javascript#comparison--switch-blocks).
I had some troubles with that some time ago and started always using the curly braces. I guess we could add the { } only if there is declarations inside the case but for consistency I would use them always even if it adds some extra clutter.